### ISSUE TITLE
Pip added as a test requirement to most things

### ISSUE THREAD

**Issue by CJ-Wright** [marcelotrevisani/grayskull#175]
It seems that many things get a pip requirement added to the tests. This seems unnecessary since most things include pip in the installing environment as it is a host dependency of most python packages. As an example https://github.com/regro/cf-graph-countyfair/blob/master/audits/grayskull/attrs_19.3.0.yml

**Comment by marcelotrevisani** [comment]
I am adding ``pip`` just in case. But the ``pip`` in test requires should not be really necessary if ``pip`` was added to the ``run`` requirements no?
or am I missing something here?

**Comment by CJ-Wright** [comment]
Yes, so I don't think we need it in the test if it is in host. But I've special cased it for the bot now.
